### ISSUE TITLE
Need to add information about new Java Active Models in the Code Architecture document

### ISSUE THREAD

**Issue by programmerdave** [LegalShield/architecture#2]
Hi @douglaslee1990,

In the java/code-architecture document, can you summarize the way you're going to handle models in the new system. Please describe how to code both Postgres and the legacy DB2 models. Basically summarize the conversations you and Jon had so that everyone else coding can look here to see how to do it.

Please let me know if you have any questions and let me know when your pull request is ready.

Thanks!

**Comment by jonstorer** [comment]
stale
